### ISSUE TITLE
F-16C JDAM Multi Drop Use is Broken in Mission

### ISSUE THREAD

**Issue by OviDiuSSwe** [Dzsek/pretense#3]
Hello i have found a issue with this mission. And its not only this mission generally speaking. But you are the only one i have found a way to contact so far to find a solution to this issue that i have found on DCS servers and missions.

Issue is when Using JDAMs in the F-16C. It can hold 4 JDAMs and you have the function where you can designate Waypoints where each JDAM should hit with the Target-Pod. And the waypoints you can use in the F-16C is WP 26-30. And in this mission 26-30 is used by Map markers. And renders the Usage of a Multi Pickle of JDAMs totally none functional. So it breaks the tactics you can use with the F-16C drastic. I have no clue if this is know already or not. But i hope for the best and enjoy the game play allot. Its just sad to see functions not be used that supposed to be there. Some what immersion broken if you get the feeling. 

Thank you for reading. Hope you understand me enough. And bet you have a solution for this. If not. I have a suggestion. But i leave that as a suggestion for later. Stay Safe & Stay Awesome !

**Comment by Dzsek** [comment]
You can quite easily overwrite points 26-30 either by manually entering coordinates from the STPT page on the DED or by creating new markpoints.

**Comment by OviDiuSSwe** [comment]
First of all thank you for taking time to make this mission and answering to issues like this.

Well the issue i have. Is that its not working like that. Its not possible to do that with the TGPD. When adding a new Waypoint the way point get overwritten to previous Waypoint due to the list of Waypoints already placed  in that range of waypoint numbers. So that renders the F-16C's usage range to a less capable plane.

The whole point of Waypoint mapping for the JDAMS with the TGPD is so your fast to designate targets on a distance. Then get closer do a popup drop on AO and evade back in to the cold. With out needing to do a full mission plan before you takeoff. Or write down coordinates on the side.

basically how i see it to solve this issue is to ether just skip 26-30 being used in the waypoint list currently in use. Or just revert the whole waypoint list to start from 699 and go backwards to 600. Then it wont interfere with any planes systems. Or do a Group system. Where you have said waypoints in groups. Like a few waypoints have 11-19 some have 31-39,41-49,51-59 you get the idea. Then you can skip the 20-30 fully. 

But that's my suggestion.

Yes its a bit "Not perfect at all" solution to make it this way. But it at least don't take away a functional system from a full fidelity Fighter plane. 

What you think.

**Comment by Dzsek** [comment]
The F-16 can store 5 markpoints, once you reach 30 its starts again from 25 and overwrites those.

If you wanna use markpoints you'll gonna overwrite some of your waypoints. Thats it.

It works perfectly well, except that you will need to re input your original waypoints from the kneeboard if you need them.

Adding an exception for the F-16 specifically is just not viable.

**Comment by OviDiuSSwe** [comment]
I don't think your understanding me correctly. The Pre made Waypoints that are placed on map for finding Locations. Like LIMA or MINE or Airfields for example. These have Waypoints programed in on the map permanently. And These Waypoints are not able to write over with TGPD. And These Waypoints reach from 1 - 49. All i'm asking is to jump over 25-30 and leave them blank. So the count would be 1-24 31-55. 

And your remark that i can Write over these Waypoints with TGPD is totally incorrect i just tested it again to be 100% on what i am writing now. I can Write them over yes. But at the same second i switch from Button 7 "Mark" to Button 4 "STPT" Said waypoints revert back to the pre set waypoints that is on server. And that results that JDAMS will not be falling on the locations i want them to explode on. They will be trying to steer towards a Town or Airfield that is miles away. That is not the target at all. So this is important to change.

The Mission is really nice to play. Its remarkably nice to be honest. But these small bugs/faults need to be fixed. Your literally condemning Systems that are really good to not be used and so your condemning a whole plane model to not be played in the long run.

**Comment by Dzsek** [comment]
If you think theres a bug with the tgp not creating markpoints report it to ED.
You can't remove waypoints in DCS from the middle of a route. it just brings the higher number waypoints down to fill the gap.

**Comment by OviDiuSSwe** [comment]
Hello ok i have talked to ED. and this is there point of view on it. 
https://forum.dcs.world/topic/330030-tgpd-waypoint-mapping-for-jdams-broken/

So question is it so that the waypoints are bound to AI movement and is vital in that way ? Other wise i would recommend just removing waypoints. More of a simulator then. But hey thx ether way for taking time to make this mission. Me and my friends have been playing it allot. And we finished it yesterday after 4-5 days of on and off playing. Its really nice and fun. Keep at it. Hope we find a solution for all this problems in the end.

**Comment by Dzsek** [comment]
Did you do the cursor zero as he describes?

I am not gonna remove the waypoints, TGP markpoints seem to work correctly for me (as long as you do the cursor zero after setting them). 
If they bother you, remove them yourself.
